src/main/java/io/leonis/zosma/ipc/io/ConfigPublisher.java

```diff
@@ -1,20 +1,35 @@
 package io.leonis.zosma.ipc.io;
 
 import com.typesafe.config.*;
+import java.time.Duration;
 import lombok.AllArgsConstructor;
 import org.reactivestreams.*;
 import reactor.core.publisher.Flux;
 
 /**
+ * The Class ConfigPublisher.
+ *
+ * This class represents a {@link Publisher} of {@link Config}.
+ *
  * @author Jeroen de Jong
  */
 @AllArgsConstructor
 public final class ConfigPublisher implements Publisher<Config> {
```

Comment:
I think we should remove this class with the dependency on typesafe.config; should we do it in a seperate PR ?